src/tactic/group.lean

```diff
@@ -0,0 +1,105 @@
+/-
+Copyright (c) 2020. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author: Patrick Massot
+-/
+import tactic
```

Comment:
This seems like a very strong import for this file, no?

Reply:
I replaced it with the minimal `import tactic.ring`.